eclass/kernel-install.eclass

```diff
@@ -66,6 +66,7 @@ _IDEPEND_BASE="
 	)
 	initramfs? (
 		>=sys-kernel/installkernel-14[dracut(-)]
+		>=sys-kernel/installkernel-14[ugrd(-)]
 	)
```

Comment:
```
|| (
	    >=sys-kernel/installkernel-14[dracut(-)]
		>=sys-kernel/installkernel-14[ugrd(-)]
)
```
We only need one of them.

Reply:
are you saying the change I made checks that you have it set to use dracut and ugrd, but I should put that || before the group so it checks that any of those are set?

Reply:
Currently portage will force both flags to be enabled at the same time yes. 

We need an or operator.